features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/RumFeature.kt

```diff
@@ -298,6 +292,20 @@ internal class RumFeature(
 
     // region Internal
 
+    private fun handleTelemetryEvent(event: Map<*, *>) {
```

Comment:
probably with that change we can send class directly instead of using `Map`? It will require a change of the check right at the beginning of `onReceive`, but it is a minor thing.

Reply:
yup exactly what @xgouchet was pointing out also, I am going to apply the changes now.